src/App.tsx

```diff
@@ -1,24 +1,202 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import Footer from './components/Footer';
+import TodoList from './components/TodoList';
+import Header from './components/Header';
 
-const USER_ID = 0;
+export const userId = 10590;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
```

Comment:
Just use empty string if there is no error
```suggestion
  const [error, setError] = useState('');
```